backend/code_review_backend/issues/api.py

```diff
@@ -93,7 +95,9 @@ class DiffViewSet(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         diffs = (
-            Diff.objects.all()
+            Diff.objects
+            # Because of the perf. hit filter issues that are not older than today - 3 months.
+            .filter(created__gte=date.today() + relativedelta(months=-3))
```

Comment:
Nit: you could use the standard Python lib here to avoid adding `dateutil` to the dependencies. It won't be precisely three months, but it should be OK for our needs: `datetime.now() - timedelta(days=90)`